src/Icon/index.js

```diff
@@ -2,17 +2,32 @@ import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native'
 import { Icon, IconToggle } from '@protonapp/react-native-material-ui'
 
+
+
 export default class WrappedIconToggle extends Component {
   render() {
-    let { iconName, iconColor, onPress } = this.props
+    let { iconName, iconColor, onPress, iconSize } = this.props
```

Comment:
So for backwards compatibility, there will be times when `iconSize` isn't defined. You'll have to make sure to cover these cases. You can do that either by using React's default props (see the article I sent you), or just by checking that iconSize is defined. It should still default to `24`. (Also, for context, just by adding the default inside the manifest doesn't set that default for currently existing apps)